ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -255,10 +266,31 @@ objects:
         sshKey: |
           ${SSH_PUB_KEY}
         EOF
+        elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
+            cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1
+        baseDomain: ${BASE_DOMAIN:-ci.azure.devcluster.openshift.com}
+        metadata:
+          name: ${CLUSTER_NAME}
+        controlPlane:
+          name: master
+          replicas: 3
+        compute:
+        - name: worker
+          replicas: 3
+        platform:
+          azure:
+            baseDomainResourceGroupName: os4-common
```

Comment:
Is this already a set name?  Something that is part of the account?

Reply:
installer currently picks the first DNS Zone from the account based on the base domain, while most cloud providers allow multiple DNS zones with the same name.

We have https://jira.coreos.com/browse/CORS-1070 already in pipeline to allow users to choose the exact Zone (without the guessing). But Microsoft people were far too un-comfortable with leaving the guessing for Azure addition and insisted we add this field to Azure platform while we were frozen for AWS.

>  Something that is part of the account?

Yes this resource group is present in the CI subscription.